assets/styl/_components/card.twig

```diff
@@ -0,0 +1,49 @@
+{# Card
+ #
+ # @param uri   isDated                (optional) (default true)
+ # @param str   theme                  (optional) override theme
+ # @param bool  lazyloadImg            (optional) image lazyload (default false)
+ # @param array classes                (optional) classes array added to each cards (defaut [])
+ # @param str   attributes             (optional) (default '')
```

Comment:
You should be consistent in your twig files, use either `string` or `str`, but not both

Reply:
haha pasted from vega